htdocs/main.inc.php

```diff
@@ -2825,12 +2837,16 @@ function top_menu_bookmark()
 	// accesskey is for Windows or Linux:  ALT + key for chrome, ALT + SHIFT + KEY for firefox
 	// accesskey is for Mac:               CTRL + key for all browsers
 	$stringforfirstkey = $langs->trans("KeyboardShortcut");
-	if ($conf->browser->name == 'chrome') {
-		$stringforfirstkey .= ' ALT +';
-	} elseif ($conf->browser->name == 'firefox') {
-		$stringforfirstkey .= ' ALT + SHIFT +';
-	} else {
+	if ($conf->browser->os === 'macintosh') {
```

Comment:
Do you mean shortcut is CTL on mac, even on firefox ?

Reply:
Hello @eldy 
yes there is no ALT key on the mac.
CTL + key is possible (shorter)
CTL + SHIFT + key is possible too